ml/notebooks/netdata_anomaly_detection_deepdive.ipynb

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
<p>Regarding the first paragraph: I would add a sentence to state very explicitly that users do not need to follow the steps in this workbook to enable ML-based anomaly detection and include a link to our docs: https://learn.netdata.cloud/docs/agent/ml#configuration</p><p>---</p><p>Just checking in on the note: Do we still want to have the e-mail address and discord channel in there? Since we're trying to move many discussions to GH discussions, it might be useful to start a thread and link it here?</p><p>---</p><p>The section where we explain what people can expect from the notebook is a good thought. I would make the following changes:</p><ul><li>Add the preparation bit as the first item. Something along the lines of: "1. Set up an environment to pre-process data with Pandas"</li><li>Refine point 5 to describe what they will learn.</li><li>Link these points to the corresponding sections (not sure if that will work as the content is in different cells.)</li></ul><p>It could work if you set and anchor with an id <code>&lt;a id="pull-data"&gt;&lt;/a&gt;</code> at the section heading and reference it in the list above: <code>[Pull some recent data](#pull-data)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netdata/netdata/pull/12313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
A little follow up now that the comments don't seem to be in their right place: 

Whenever a comment just contains a `Line #n Bla bla`  I reviewed a comment that was within a code cell. I think the only change I made in these comments so far was to write out abbreviations. Like when you wrote dims, I replaced it with dimensions.

Reply:
added a few things in for this.